libsolidity/formal/CHC.cpp

```diff
@@ -27,6 +27,10 @@
 
 #include <libsolidity/ast/TypeProvider.h>
 
+#include <libsolutil/Algorithms.h>
+
+#include <queue>
```

Comment:
This is unused in this latest version, isn't it?

Reply:
Removed